mappings/net/minecraft/client/settings/GameSettings.mapping

```diff
@@ -40,7 +45,36 @@ CLASS none/bdt net/minecraft/client/settings/GameSettings
 		FIELD x VBO Lnone/bdt$a;
 		FIELD y TOUCHSCREEN Lnone/bdt$a;
 		FIELD z CHAT_SCALE Lnone/bdt$a;
+		METHOD <init> (Ljava/lang/String;ILjava/lang/String;ZZ)V
+			ARG 0 key
+			ARG 1 isSlider
+			ARG 2 isBoolean
+		METHOD <init> (Ljava/lang/String;ILjava/lang/String;ZZFFF)V
+			ARG 0 key
+			ARG 1 isSlider
+			ARG 2 isBoolean
+			ARG 3 sliderMin
+			ARG 4 sliderMax
+			ARG 5 sliderStep
+		METHOD a isSlider ()Z
+		METHOD a setMaximum (F)V
+			ARG 0 value
+		METHOD a getOption (I)Lnone/bdt$a;
+			ARG 0 id
+		METHOD b isBoolean ()Z
```

Comment:
Well if this corresponds to the field of the same name ... the field can't be named `boolean`. And I think switch is the closest thing I can think of that represents an on/off state (other than checkbox but that's usually rendered in a specific way).